backend/app/services/group_service.py

```
@@ -482,21 +484,92 @@ def add_member(
     }
-
     return GroupMemberResponse(**response_data)
 
 
+def _get_group_or_404(db: Session, group_name: str) -> Namespace:
+    """Get an active group or raise 404."""
+    group = (
+        db.query(Namespace)
+        .filter(
+            Namespace.name == group_name,
+            Namespace.is_active == True,
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)